diff -Nuard xosd-2.0.0.orig/src/libxosd/xosd.c xosd-2.0.0/src/libxosd/xosd.c
--- xosd-2.0.0.orig/src/libxosd/xosd.c	Mon Oct 21 02:38:53 2002
+++ xosd-2.0.0/src/libxosd/xosd.c	Tue Oct 29 19:44:00 2002
@@ -34,7 +34,7 @@
 #include <X11/extensions/shape.h>
 #include <X11/Xatom.h>
 
-#include "xosd.h"
+#include "../xosd.h"
 
 #define DEBUG(args...) /* fprintf (stderr, "%s: %s: %d: ", __FILE__, __PRETTY_FUNCTION__, __LINE__); fprintf(stderr, args); fprintf(stderr, "\n") */
 
diff -Nuard xosd-2.0.0.orig/src/xmms_plugin/xmms_osd.c xosd-2.0.0/src/xmms_plugin/xmms_osd.c
--- xosd-2.0.0.orig/src/xmms_plugin/xmms_osd.c	Mon Oct 21 02:39:17 2002
+++ xosd-2.0.0/src/xmms_plugin/xmms_osd.c	Tue Oct 29 19:44:16 2002
@@ -29,7 +29,7 @@
 #include <xmms/xmmsctrl.h>
 #include <xmms/configfile.h>
 
-#include <xosd.h>
+#include "../xosd.h"
 
 #define DEBUG(a) /*fprintf (stderr, "%s: %s: %d: %s\n", __FILE__, __PRETTY_FUNCTION__, __LINE__, a)*/
 
