--- xosd-2.2.12/configure.ac.orig	2004-09-14 03:05:42.000000000 +0200
+++ xosd-2.2.12/configure.ac	2006-08-26 23:41:02.159022500 +0200
@@ -48,7 +48,7 @@
 sed -e 's/^build_old_libs=yes/build_old_libs=no/' libtool > libtool-disable-static
 chmod +x libtool-disable-static
 
-LIBS="$LIBS $X_PRE_LIBS $X_LIBS $X_EXTRA_LIBS"
+LIBS="$LIBS $X_LIBS $X_EXTRA_LIBS"
 CFLAGS="$CFLAGS $X_CFLAGS -Wall"
 M4DATADIR="$datadir/aclocal"
 AC_SUBST(M4DATADIR)
@@ -60,7 +60,6 @@
 	     AC_MSG_ERROR([*** X11 not threadsafe ***]))
 AC_CHECK_LIB(Xext, XShapeQueryExtension,,
 	     AC_MSG_ERROR([*** X11 Shape extension not found ***]))
-AC_CHECK_LIB(Xt, main)
 AC_CHECK_LIB(pthread, pthread_create,,
 	     AC_MSG_ERROR([*** POSIX thread support not found ***]))
 
@@ -95,8 +94,8 @@
                                     AC_DEFINE(HAVE_XINERAMA,1,[Define this if you have libXinerama
 installed])
                                     ac_have_xinerama="yes"],,
-                                    [$X_LIBS $X_PRE_LIBS -lXext $X_EXTRA_LIBS]),
-                     [$X_LIBS $X_PRE_LIBS -lXext $X_EXTRA_LIBS])
+                                    [$X_LIBS -lXext $X_EXTRA_LIBS]),
+                     [$X_LIBS -lXext $X_EXTRA_LIBS])
 fi
 
 
